common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockTransferor.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.spark.network.shuffle;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Not sure why linter did not catch this - move it to the third party section of imports. (along with existing com.google)

Reply:
done